### ISSUE TITLE
Gif will not work

### ISSUE THREAD

**Issue by LochNessLobster** [shrek-uberr/genbread.github.io#1]
Gif will not open on PC or in Discord client.

**Comment by genbread** [comment]
yeah im trying to recreate the old shrek-uber thing that got deleted, i copied the exact code and shit and its not working so either they patched it or im missing something. not really worth my time but the video itself still crashes discord if you want that. the original gif was also blank, btw
